### PR TITLE
Add RC Release Procedure

### PR DESCRIPTION
**Pull request by t-kikuc**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Does this PR introduce a user-facing change?**:

- **How are users affected by this change**:
- **Is this breaking change**:
- **How to migrate (if breaking change)**:

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -83,3 +83,35 @@ A bugfix for a functional issue (not a data loss or security issue) that only af
...
+            output(from newer to older):
+
+            ```
+            Add reference to the blog that shows how to install control plane on ECS (#4746)
+            Update copyright (#4745)
+            ...
+            Add docs for SCRIPT_RUN stage (#4734)
+            ```
+
+         2. Classify the changes into 'Notable Changes' and 'Internal Changes'.
```
Comment (ffjlabo) [inline RELEASES.md:114]: [nit] How about adding the explanation for 'Notable Changes' and 'Internal Changes' 👀 
- Notable Changes
    - Changes that should notify users (i.g., new features, breaking changes, etc.)
- Internal Changes
    - Changes that don't affect the functions

@khanhtc1202 This is my understanding. Are you aware of it? 👀

**Review by ffjlabo — COMMENTED** [review]
Nice catch 👍 Added a nit comment.

File: RELEASES.md (hunk 1 above)
Comment (khanhtc1202) [reply RELEASES.md:114]: I think those words have mean by itself. But if you want to add definition for those, should be added at other level in this release note file. Say, place of defining versions semantic or such 👀

Comment (ffjlabo) [reply RELEASES.md:114]: > I think those words have mean by itself.

@khanhtc1202 
Oh I see, I got it. That's right (checked with translator🙏 )
I agree with it 👍 We don't need do that. Thanks!

Comment (ffjlabo) [reply RELEASES.md:114]: @t-kikuc Plz forget to the nit comment 🙏

**Review by ffjlabo — APPROVED** [review]
🚀

**Review by khanhtc1202 — COMMENTED** [review]
🤖

**Review by khanhtc1202 — APPROVED** [review]

**Pull request merged**